### PR TITLE
zh_CN.lang

### PR DESCRIPTION
**Pull request by DYColdWind**

### CODE REVIEW

File: src/main/resources/assets/yoyos/lang/zh_CN.lang [hunk 1]
```diff
@@ -22,7 +27,7 @@ info.weight.name=重量：%s
 info.duration.name=持续时间：%s秒
 info.length.name=长度：%s个方块
 
-info.sticky.name=具有粘液性的悠悠球会黏着在方块上持续滚动。
+info.sticky.name=具有粘液性的悠悠球会黏着在方块上持续旋转。
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/lang/zh_CN.lang]: 粘性 or 粘液性？

File: src/main/resources/assets/yoyos/lang/zh_CN.lang [hunk 2]
```diff
@@ -22,7 +27,7 @@ info.weight.name=重量：%s
...
 info.sticky.retraction.name=保持%s以收回悠悠球。
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/lang/zh_CN.lang]: We usually use “按住”
and “缩回” / “收短” may be better?

File: src/main/resources/assets/yoyos/lang/zh_CN.lang [hunk 3]
```diff
@@ -70,6 +75,9 @@ modifier.sticky.desc=§o我...我无法理解§r\n黏着在方块上，从而带
 modifier.gluey.name=凝胶
 modifier.gluey.desc=§o安全，无毒§r\n黏着在方块上，从而带动玩家进行远程移动；但会损耗耐久。
 
+modifier.collecting.name=收集
+modifier.collecting.desc=§o无人入眠！§r\n抓取并收集所有途中经过的掉落物。
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/lang/zh_CN.lang]: "一并抓住" / “一网打尽” Still cannot find a perfect translation

Comment (Snownee) [reply src/main/resources/assets/yoyos/lang/zh_CN.lang]: BTW in line 73 "get off" -> "逃脱"
in line 55 "运转" -> "转动"

File: src/main/resources/assets/yoyos/book/zh_cn/modifiers/collecting.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+  "modifier": "collecting",
+  "text": [
+    {
+      "text": "漏斗真的很适合收集东西，我真的想把它与我的悠悠球融合..."
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/book/zh_cn/modifiers/collecting.json]: Where's the "guess"...
"或许我可以将二者结合……"

File: src/main/resources/assets/yoyos/lang/zh_CN.lang [hunk 5]
```diff
@@ -5,14 +5,19 @@ item.yoyos.diamond_yoyo.name=钻石质悠悠球
...
 item.yoyos.cord.name=捆绳
 
+enchantment.collecting=收集
+
 yoyos.config.vanilla=原版悠悠球
 yoyos.config.enable=启用
 yoyos.config.tinkers=匠魂悠悠球
 yoyos.config.swing=悠悠球摆动
 
+yoyos.config.materials=材料参数设置
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/lang/zh_CN.lang]: 可 配置/设置 的材料 参数/属性

File: src/main/resources/assets/yoyos/lang/zh_CN.lang (hunk 3 above)
Comment (DYColdWind) [reply src/main/resources/assets/yoyos/lang/zh_CN.lang]: ‘BTW in line 73 "get off" -> "逃脱"’
逃脱？I think it's better to use '移动' here
His main purpose is to move, not to run away.
And running away is also part of the move
In addition, others I have modified（另外，其他的我已经进行完善）

**Review by Snownee — APPROVED** [review]

**Comment by Jozufozu** [comment]
Thanks again you two.

**Pull request merged**